aws_lambda_builders/workflows/go_modules/utils.py

```diff
@@ -0,0 +1,30 @@
+"""
+Commonly used utilities
+"""
+
+import os
+import subprocess
+
+
+class OSUtils(object):
+    """
+    Wrapper around file system functions, to make it easy to
+    unit test actions in memory
+    """
+    @property
+    def environ(self):
+        return os.environ
```

Comment:
We should copy this (`os.envrion.copy()`), otherwise we risk changing Env Vars on the machine.

Reply:
Fixed